gateway/cli/src/main.rs

```diff
@@ -41,17 +39,6 @@ pub enum Commands {
         #[clap(long)]
         federation_id: FederationId,
     },
-    /// Deposit funds into a gateway federation
-    Deposit {
```

Comment:
`Deposit` is no longer needed since `deposit_address` will spawn a state machine that will take care of getting the txoutproof and the raw transaction.